docker/hadoop/cdh/Dockerfile

```diff
@@ -27,28 +29,17 @@ ENV DISTRIBUTION='cdh' \
     HIVE_PACKAGE=${HIVE_PACKAGE:-hive} \
     HIVE_HOME=/usr/lib/${HIVE_PACKAGE}
 
+COPY ${PATH_PREFIX}/scripts/sbin ${PATH_PREFIX}/../common/sbin scripts/install_python_version /usr/sbin/
 RUN \
```

Comment:
@honzasterba should the RUN command be executed with the other following 2?

the apt cache clean-up is imho only done by install_python_version

    apt-get clean
    rm -rf /var/cache/apt/*

Are we keeping the caches in the layers created for each RUN?